### PR TITLE
Chore: Housekeeping module name references

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

Housekeeping PR.

- [x] Updates from `theme-b2c-tailwind` to `styles` package.
- [x] Makes `@faststore/ui` storybook use the `styles` package.
- [x] Fixes plop style generator.
- [x] Updates some `README.md` and docs. 
- [x] Removes unused `@vtex/tsconfig` from `@faststore/ui`.

## References
https://github.com/vtex/faststore/pull/1326
https://github.com/vtex/faststore/pull/1379
https://github.com/vtex/faststore/pull/1296

### CODE REVIEW

**Review by filipewl — APPROVED** [review]
Nice!

File: generators/plopfile.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default (plop: NodePlopAPI) => {
         },
         {
           type: 'add',
-          path: '../themes/theme-b2c-tailwind/src/{{atomicGroup}}/{{kebabCase name}}.css',
+          path: '../packages/styles/src/{{atomicGroup}}/{{kebabCase name}}.css',
```
Comment (hellofanny) [inline generators/plopfile.ts:33]: Cool! I thought about it yesteday ~ and now it's updated 💯

**Review by hellofanny — APPROVED** [review]
Niceeeeee 💯 THANK YOU.

File: generators/plopfile.ts (hunk 1 above)
Comment (gvc) [reply generators/plopfile.ts:33]: :ant: mind reader :eye:

**Pull request merged**